packages/react-static/src/utils/binHelper.js

```diff
@@ -116,15 +116,15 @@ ignoredExtensions.forEach(ext => {
 
 const originalConsoleError = console.error
 console.error = (...args) => {
-  if(args.length === 0){
+  if (args.length === 0) {
```

Comment:
I think this is just my prettier config - I can back this change out if you want